### ISSUE TITLE
Duna is rotating impossibly fast

### ISSUE THREAD

**Issue by SelectricSimian** [KSP-64k/64k#5]
I understand that Duna and Ike's rotational periods have been adjusted to ensure that they are tidally locked.  However, everything does not scale nicely from the stock Kerbol system, and setting up Duna's orbital period in this way makes it rotate much, much faster than is physically possible.  I learned this the hard way, because it turns out that this anomaly also makes landing on Duna impossible.

`RealSolarSystem.cfg` gives Duna [a radius of `2048000` m](https://github.com/KSP-64k/64k/blob/master/RealSolarSystem/RealSolarSystem.cfg#L329), and [a rotational period of approximately `5086` s.](https://github.com/KSP-64k/64k/blob/master/RealSolarSystem/RealSolarSystem.cfg#L331)

Duna's circumference is therefore `2048000 m * 2π = 1.2868E+7 m`.  This means that Duna's surface is moving at `1.2868E+7 m / 5086 s = 2530.08 m/s`, at the equator.

**You read that correctly.  Just "sitting still" on Duna's equator, you would be moving at 2.5 _kilometers_ per second.**

By comparison, circular orbital velocity for 0 m above Duna's surface is `2.45 km/s` (calculated using the handy dandy [circular orbital velocity calculator](http://orbitsimulator.com/formulas/vcirc.html)).  **Duna's equator is rotating faster than orbital velocity.**

In other words, 64k's Duna is more like Krag's "Inaccessible" planet than stock Duna.  Fixing this requires breaking Duna and Ike's tidal locking, or moving them farther apart than is proportional to stock.  This deviation from stock is obviously not ideal, but the current situation is clearly untenable.  Not only does it make no physical sense (Duna would fly apart), it's virtually game-breaking for landing on Duna.

**Comment by Paul-kingtiger** [comment]
Yep that's just plain wrong!
I've been playing around with the config all morning, using a kepler orbit calculator to check the values.  It's made harder by the fact that KSP doesn't show Orbital period in game, which would be useful to double check the calculations.

Anyway I've recalculated everything and Duna now has a Rotational Period of ~44.5 hours, is approximately tidally locked to Ike, which is also tidally locked.

Fixed in new release 1.0.90.3
